Apps/W1/HybridGP/app/src/pages/GPMigrationConfiguration.Page.al

```diff
@@ -0,0 +1,512 @@
+                }
+            }
+
+            group(SettingsList)
+            {
+                Caption = 'Per Company';
+
+                part("GP Company Additional Settings List"; "GP Company Add. Settings List")
+                {
+                    Caption = 'Configure individual company settings';
+                    ShowFilter = true;
+                    ApplicationArea = All;
+                    UpdatePropagation = Both;
+                }
+            }
+        }
+    }
+
+    actions
+    {
+        area(Processing)
+        {
+            action(ResetAllAction)
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = 'Reset Defaults';
+                ToolTip = 'Reset all companies to the default settings.';
+                Promoted = true;
+                PromotedCategory = Process;
+                Image = Setup;
+
+                trigger OnAction()
+                begin
+                    if Confirm('Are you sure? This will reset all company migration settings to their default values.') then
+                        ResetAll();
+                end;
+            }
+
+            action(SetDimensions)
+            {
+                ApplicationArea = Basic, Suite;
+                Caption = 'Set All Dimensions';
+                ToolTip = 'Attempt to set the Dimensions for all Companies.';
+                Promoted = true;
+                PromotedCategory = Process;
+                Image = Dimensions;
+
+                trigger OnAction()
+                var
+                    GPPopulateDimensionsDialog: Page "GP Set All Dimensions Dialog";
+                    SelectedDimension1: Text[30];
+                    SelectedDimension2: Text[30];
+                    BlanksClearValue: Boolean;
+                begin
+                    GPPopulateDimensionsDialog.RunModal();
+                    if GPPopulateDimensionsDialog.GetConfirmedYes() then begin
+                        SelectedDimension1 := GPPopulateDimensionsDialog.GetDimension1();
+                        SelectedDimension2 := GPPopulateDimensionsDialog.GetDimension2();
+                        BlanksClearValue := GPPopulateDimensionsDialog.GetBlanksClearValue();
+
+                        if (SelectedDimension1 <> '') or BlanksClearValue then
+                            AssignDimension(1, SelectedDimension1);
+
+                        if (SelectedDimension2 <> '') or BlanksClearValue then
+                            AssignDimension(2, SelectedDimension2);
+                    end;
+                end;
+            }
+        }
+    }
+
+    procedure ShouldShowIntroductionNotification(shouldShow: Boolean)
+    begin
+        ShowIntroductionNotification := shouldShow;
+    end;
+
+    procedure ShouldShowManagementPromptOnClose(shouldShow: Boolean)
+    begin
+        ShowManagementPromptOnClose := shouldShow;
+    end;
+
+    trigger OnInit()
+    begin
+        ShowIntroductionNotification := true;
```

Comment:
This is not a good UX pattern. We should not show notifications on each opening of the page.

Reply:
Removed the notification